### DISCUSSION TITLE
Can the AI Deck Infrastructure support running linear regression models?

### DISCUSSION THREAD

**Discussion by usernamemanr** [bitcraze/discussions#1130] [Q&A]
Context: Hi, a student here, new to AI-deck. I understand that AI deck is primarily designed for CNNs. However, I see that it is capable of accepting Python libraries.

So, my question is if the infrastructure are able to handle linear regression models from Pickle or Joblib?

**Comment by knmcguire** [comment]
Currently it is not possible to run anything other than C libraries directly on the AIdeck. Python libraries can be used to train the network beforehand though, such as Pytorch and such. 

For these capabilities we advise to contact [Greenwaves technologies](https://greenwaves-technologies.com/) as they are maintaining the SDK for the aideck we use

**Reply by usernamemanr** [reply]
I see, thank you for the response
